test/dependencies.jl

```diff
@@ -184,7 +184,7 @@ end
             # Since Julia v1.9 Zlib_jll doesn't have logs directory at all
             @static if VERSION < v"1.9-DEV"
                 zlib_log_files = if os(platform) == "macos"
-                    ["Zlib.log.gz", "fix_identity_mismatch_libz.1.2.11.dylib.log.gz", "ldid_libz.1.2.11.dylib.log.gz"]
+                    ["Zlib.log.gz", "fix_identity_mismatch_libz.1.2.12.dylib.log.gz", "ldid_libz.1.2.12.dylib.log.gz"]
```

Comment:
Why did you change this?  How did it work before?

Reply:
I changed it because this test failed. I don't know how it worked before – I assume the test was written when `libz @1.2.11` was current.

Reply:
I was very confused about why this didn't fail in CI (nor it failed for me recently), but this code is run only on macOS (and running Docker in macOS in CI is a mess: #133).  I guess the change is fine then.